### ISSUE TITLE
Base plots or ggplot2?

### ISSUE THREAD

**Issue by danilofreire** [bquast/rddtools-paper#1]
Hello Bastiaan and Mathieu!

Congratulations for the great package. I've been using the "rddtools" for quite some time and I'm happy to see that it will (hopefully) appear in the R Journal. My question is: why not settle down with either base plot or ggplot2? Using one of them would be more consistent. Also, and only if possible, it would be nice to have graphs that are almost publication-ready. 

In the same vein, a friend-of-a-friend of mine wrote a nice post on graphical representations of RDDs. It may be useful to you guys. The code is available online, too: [http://thepoliticalmethodologist.com/2015/03/10/graphical-presentation-of-regression-discontinuity-results/](http://thepoliticalmethodologist.com/2015/03/10/graphical-presentation-of-regression-discontinuity-results/).

Best luck with the paper!

**Comment by bquast** [comment]
agreed, this paper has to be in some kind of state soon, its a priority for now but I will look at that asap

**Comment by bquast** [comment]
I'm filing it with the package itself, feel free to add to it:

https://github.com/bquast/rddtools/issues/1

**Comment by danilofreire** [comment]
Thanks a lot for taking my suggestion in consideration. That was the only
thing that I thought should be improved. If you can also add Natália's
graphs to RDDtools it would be great too, but it's surely not a priority. I
send you a message if I find anything that can be included in the package.
